linera-service/src/linera/main.rs

```diff
@@ -1039,12 +1039,32 @@ impl Runnable for Job {
                 );
             }
 
+            ListEventsFromIndex {
+                chain_id,
+                stream_id,
+                start_index,
+            } => {
+                let context = ClientContext::new(
+                    storage.clone(),
+                    options.inner.clone(),
+                    wallet,
+                    Box::new(signer.into_value()),
+                );
+                let start_time = Instant::now();
+                let chain_id = chain_id.unwrap_or_else(|| context.default_chain());
+                let index_events = storage
+                    .list_events_from_index(&chain_id, &stream_id, start_index)
+                    .await?;
+                println!("{:?}", index_events);
```

Comment:
We could pretty print this:

```suggestion
                println!("{:#?}", index_events);
```

…or even output it as a table with one event per line?

Reply:
Ok.